### ISSUE TITLE
Show "Execute JavaScript" status in "Download" block within "ForEach" block

### ISSUE THREAD

**Issue by anewuser** [pipes-digital/pipes#106]
When you add a  `Download` block to a `ForEach` block, there's no indication whether `Execute JavaScript` is enabled or not:

![pipes](https://user-images.githubusercontent.com/6733770/181945153-74e0e695-94a2-42e9-a4ba-0e32eaf320ca.png)

**Comment by onli** [comment]
Right. Problem is: It really won't, the download block with the checkbox ceases to exist. What feels better: A checkbox in the forEach block, or that the javaScript checkbox state is moved into the forEach block (but then stays static) when they are merged?

**Comment by anewuser** [comment]
I was just thinking of a grayed-out label to confirm that JS execution was disabled in a loop. 

While testing a pipe days ago, I had the impression that a loop *did* execute JS at least once when I skimmed over `view output`, even though I hadn't enabled JS in the "wrapped" `Download` block. 

I couldn't investigate this further because the block was downloading multiple articles from the same feed, and it ended up triggering a DoS protection mechanism. This was one of the reasons why I suggested #99 too. 

After rewriting the pipe to download only a single article, I haven't noticed anything unusual with JS again, so I can't confirm that it ever happened.

**Comment by anewuser** [comment]
I was just thinking of a grayed-out label to confirm that JS execution was disabled in a loop, but adding a working checkbox might be useful. That way, people won't have to recreate the whole loop just to toggle JS.
